lib/displayutils.js

```diff
@@ -16,7 +16,18 @@ function resultDisplay(id, prefix, result) {
   }
 
   var line = parts.join(' - ');
-  return (result.skipped ? 'skip ' : (result.passed ? 'ok ' : 'not ok ')) + id + ' ' + line;
+  var status;
+  if (result.skipped) {
+    status = 'skip ';
+  } else if (result.passed && !result.todo) {
+    status = 'ok ';
+  } else if (!result.passed && result.todo) {
+    status = 'todo ';
```

Comment:
I'm curious, is this supposed to follow TAP? If so, the directives should actually be after `ok/not ok`.